java/src/jmri/jmrix/JmrixSystemsBundle_ca.properties

```diff
@@ -0,0 +1,64 @@
+# JmrixSystemsBundle.properties
+#
+# Revision $Revision$
+#
+# Default properties for the jmri.jmrix Systems menu
+#
+# Italian translation: Joan de Castro (joan276dca@yahoo.es) 16/08/2016
+#
+
+MenuTools = Sistema
+
+# Protocols that dont need their own file (yet)
+# share entries in this one
+
+MenuSystems = Sistema
+
+
+MenuItemAcela = Acela
+MenuItemBachrus = Bachrus
+MenuItemCAN = CAN
+MenuItemCBUS = CBUS
+MenuItemCMRI = CMRI
+MenuItemDirect = DirectDrive
+MenuItemEasyDCC = EasyDCC
+MenuItemEcos = ECoS
+MenuItemJMRIClient = JMRI Client di Rete
+MenuItemLocoNet = LocoNet
+MenuItemMRC = Model Rectifier Corp (MRC)
+MenuItemMaple = Maple Systems
+MenuItemNCE = NCE
+MenuItemOakTree = Oak Tree
+MenuItemOpenLCB = OpenLCB
+MenuItemSPROG = SPROG
+MenuItemTMCC = TMCC (Lionel)
+MenuItemSECSI = SECSI
+MenuItemSRCP = SRCP
+MenuItemWangrow = Wangrow
+MenuItemXPressNet = XPressNet
+MenuItemXPA = XPA
+MenuItemZimo = Zimo
+
+MenuItemCommandMonitor = Monitor de Comandes
+MenuItemTrafficMonitor = Monitor de Transit
+MenuItemSendCommand = Envia Comanda
+MenuItemSendFrame = Envio FMarc
+MenuItemFirmwareUpdate = Actualitza Firmware
+MenuItemSlotMonitor = Monitor de Slot
+MenuItemShowCabs = Visualizza Cabs
+MenuItemMacroCommand = Envia Macro
+MenuItemMacroEdit = Edita Macro
+MenuItemConsistEdit = Modifica Multi-Tracció
+MenuItemTrackPacketMonitor = Analitza Paquet DCC
+MenuItemClockMon = Monitor de Rellotge
+MenuItemConfigNodes = Configura Nodes
+MenuItemConsole = Consola
+MenuItemDiagnostics = Arrenca Diagnòstics
+MenuItemAssignments = Visualitza Assignacions
+MenuItemConfigTool = Configura les Connessions del Bus
+MenuItemEventTable = Taula d'Esdeveniments
```

Comment:
Also with NetBeans?


El 27/08/2016 a les 9:23, Egbert Broerse ha escrit:
>
> In java/src/jmri/jmrix/JmrixSystemsBundle_ca.properties 
> <https://github.com/JMRI/JMRI/pull/1803#discussion_r76512051>:
>
> > +MenuItemSendCommand = Envia Comanda
> > +MenuItemSendFrame = Envio FMarc
> > +MenuItemFirmwareUpdate = Actualitza Firmware
> > +MenuItemSlotMonitor = Monitor de Slot
> > +MenuItemShowCabs = Visualizza Cabs
> > +MenuItemMacroCommand = Envia Macro
> > +MenuItemMacroEdit = Edita Macro
> > +MenuItemConsistEdit = Modifica Multi-Tracció
> > +MenuItemTrackPacketMonitor = Analitza Paquet DCC
> > +MenuItemClockMon = Monitor de Rellotge
> > +MenuItemConfigNodes = Configura Nodes
> > +MenuItemConsole = Consola
> > +MenuItemDiagnostics = Arrenca Diagnòstics
> > +MenuItemAssignments = Visualitza Assignacions
> > +MenuItemConfigTool = Configura les Connessions del Bus
> > +MenuItemEventTable = Taula d'Esdeveniments
>
> To put a single quote into a translation, you have to write 2 of 
> these, like |d''Esdeven...|
> The (first) single quote is a so called escape character in properties 
> files, treating the second as e text char.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/JMRI/JMRI/pull/1803/files/1bed967ad305e7b94bee89ed3a879fa6a9154bb7#r76512051>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AUET1UBP8VJB8561Gn0AmSzlvrxzQsdFks5qj-XdgaJpZM4Jlur4>.
>